addons/web/static/src/views/fields/properties/property_definition.scss

```diff
@@ -29,6 +29,10 @@
             height: 20px;
         }
 
+        .o_field_property_dropdown > img {
+            margin-right: -18px;
```

Comment:
Meh, negative margins ^^

Reply:
the bottom border it set on the input, and I can't move the image inside of the `<input/>` :/

The only other solution I have in mind is to add CSS to remove the border of the input, and add it on a parent element, but it will require even more code (to change the color on focus, etc)